sdk/script/management-sdk.ts

```diff
@@ -169,6 +169,11 @@ class AccountManager {
         return this.post(urlEncode `/apps/${appName}/deployments/`, JSON.stringify(deployment), /*expectResponseBody=*/ true)
             .then((res: JsonResponse) => res.body.deployment);
     }
+    
+    public clearDeployment(appName: string, deploymentName: string): Promise<void> {
```

Comment:
No, I don't think they are semantically equivalent.

A CodePush deployment is an update channel (or "environment").  It has its own descriptors and relationships.

As of this change, deployment history is an erasable log of updates released to that channel.

A deployment can have zero to many associated updates.  In other words, a deployment can have deployment history or no deployment history at all, such as after clearing deployment history.  However, clearing deployment history does not change the deployment itself (its existence, its name, its deployment key, etc.).  The channel is still there and usable but waiting for a new release.

This feature erases deployment history.

>>  The deployment is essentially just a history of releases

I think that's a bit of an oversimplification, and at a high-level you may be right.